build-tools/package.json

```diff
@@ -102,7 +103,8 @@
 		"overrides": {
 			"json5@<1.0.2": "^1.0.2",
 			"json5@>=2.0.0 <2.2.2": "^2.2.2",
-			"qs": "^6.11.0"
+			"qs": "^6.11.0",
+			"@oclif/core": "~2.4.0"
```

Comment:
This shouldn't be needed. If we need to use a different version of oclif then let's update the dependencies of the individual packages.

Reply:
Should we down grade build-tool to depend on ~2.4.0 instead of ^2.8.6?